Assets/Scripts/Managers/GameLoopManager.cs

```diff
@@ -26,7 +26,7 @@ void Start()
     void Update()
     {
         UpdateTurn();
-        Debug.Log("playerturn" + isPlayerTurn);
+        //Debug.Log("playerturn" + isPlayerTurn);
```

Comment:
I'd prefer to keep this in until we have functional cards working, since it's a player action that takes time it's going to for sure need some debugging